src/plugins/files/server/integration_tests/file_service.test.ts

```diff
@@ -69,7 +71,10 @@ describe('FileService', () => {
       },
       http: {},
     });
+
     esClient = coreStart.elasticsearch.client.asInternalUser;
+
+    assert.strictEqual(await esClient.ping(), true, 'Unable to reach ES, test setup failed!');
```

Comment:
If I understood the investigation correctly, I think that the test has been stuck at 

```
await kbnRoot.setup();
     coreStart = await kbnRoot.start();
```

so this new assertion anyway would not have run and wouldn't not log anything? or am I missing something?